test/pthread/.gitignore

```diff
@@ -0,0 +1 @@
+*~
```

Comment:
Quick answer, we don't.  It is taken out in my next commit.

Long answer is that emacs creates a file like `editedFile.ext~` if you modify and save `editedFile.ext`.  This gives you a chance to revert your changes to the original if you screw things up.  I just added this to take them out of my `git status` view.  I usually add files manually to a commit, not as a group, so it isn't really a problem to get rid of this.